app/src/main/java/com/newgogiyumandroid/MainActivity.kt

```diff
@@ -18,47 +29,61 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val thread = NetworkThread()
-        thread.start()
-
+        if (checkInternetConnection()) {
+            readData();
+        } else {
+            progressImg.visibility = View.GONE
+        }
     }
 
-    inner class NetworkThread: Thread() {
-        override fun run() {
-
-            val jsonArray = readData()
-
-            for (i in 0 until jsonArray.length()) {
-                val jsonObject = jsonArray.getJSONObject(i)
-
-                val imageUrl:String = jsonObject.getString("image")
-                val name = jsonObject.getString("name")
-                val ename = jsonObject.getString("e_name")
-
-                val foodList = FoodList(imageUrl, name, ename)
-                data.add(foodList)
+    fun readData() {
+        val client = OkHttpClient.Builder().build()
+        val req = Request.Builder().url("https://gogiyum.com/api/menu").build()
```

Comment:
Can you change hard coded variable to static variable?

Reply:
> Can you change hard coded variable to static variable?

what this mean? can you explain it more detailed to me?

Reply:
```
private static final String MENU_URL = "https://gogiyum.com/api/menu";
```
Then put with variable on the top.

Reply:
Okay